src/components/Form/Form.tsx

```diff
@@ -8,16 +9,26 @@ interface IState{
   } | undefined
 }
 
-const Form:React.FC <IState> = ({ questions }) => {
+const Form:React.FC <IState> = ({ questions, currentCountry}) => {
+  const [answer,setAnswer] = useState<string | number>('')
```

Comment:
I wonder if we should hold the answer in a state hook in the QuizPage and send a prop to form that setsAnswer. That way, upon rendering the Answer component we can just send the answer from QuizPage to Answer component.